scripts/contracts-sync.sh

```diff
@@ -11,6 +11,19 @@ rsync -av --progress lsp-smart-contracts/docs/. ../docs/contracts/
 cd ..
 rm -rf tmpDocsSync
 
-# Add Specs Links to the end of each file
-for FILE in docs/contracts/contracts/**/*.md; do cat $FILE docs/contracts/_links.md > $FILE.copy; done 
-for FILE in docs/contracts/contracts/**/*.md.copy; do cp $FILE `echo $FILE | sed 's/.copy//g'` && rm $FILE; done
+# Add Global Links to the end of each file from `docs/contracts/_links.md`
+linksFile="$(awk '$0 == "<!-- GLOBAL LINKS -->" {i=1;next};i' docs/contracts/_links.md)"
+for FILE in docs/contracts/contracts/**/*.md;
+do
+fileToAddLinks="$(awk '$0 == "<!-- GLOBAL LINKS -->" {i=1;next};i' $FILE)"
+if [ "$linksFile" != "$fileToAddLinks" ];
+then 
+printf '%s\n' '/<!-- GLOBAL LINKS -->/,$d' w | ed -s $FILE
+cat $FILE docs/contracts/_links.md > $FILE.copy
+cp $FILE.copy $FILE && rm $FILE.copy;
+else echo "up to date"
+fi; 
+done
+
+# Prettify the auto-generated folder
+npx prettier -w docs/contracts/
```

Comment:
This is probably what caused the change from `symbol_` to `symbol*`.
We might need a `.prettierrc` file in the repo with some config to avoid that.

Reply:
It happened in the past, same issue, you have to escape `_` this character. We def need to look into a fix. If we don't find I can remove prettier as it is not important.

Reply:
to add basically a space before the links, but I thing it is overkill. I will remove it